_data/library/versions.yml

```diff
@@ -0,0 +1 @@
+["0.6.0","0.6.1","0.6.2","0.6.3","0.6.4","0.6.5","0.6.6"]

```

Comment:
Can this be written like this instead?
```yml
- 0.6.0
- 0.6.1
- 0.6.2
- 0.6.3
- 0.6.4
- 0.6.5
- 0.6.6
```
I think it's the same YAML value in both cases, isn't it? That would give nicer diffs in subsequent additions.